pkg/kube/pod_writer_test.go

```diff
@@ -37,14 +37,15 @@ var _ = Suite(&PodWriteSuite{})
 
 func (p *PodWriteSuite) SetUpSuite(c *C) {
 	var err error
+	contxt := context.TODO()
```

Comment:
@viveksinghggits here I have used `contxt `as we are initializing `ctx` with a timeout at line 65